CGPs/cgp-0042.md

```diff
@@ -61,6 +61,8 @@ In order to verify the proposal, the address of `stableToken` in the `ExchangeBR
 
 After doing that, it can be verified [using the tooling](https://docs.celo.org/community/release-process/smart-contracts#verify-proposed-release-candidate). A local tag should be created at commit TODO at [luisgj fork of monorepo](https://github.com/luisgj/celo-monorepo/tree/rc/v6) at the branch named `rc/v6`. The tag created is the one that should be used as the `-b` flag (set as `$RELEASE_CANDIDATE` in the documentation).
 
+The cli version required to verify this proposal been released yet, but it can be found [in this branch](https://github.com/luisgj/celo-monorepo/tree/cREAL_cli). After pulling, it can be built by executing `yarn && yarn build` on the root, and then the binary for the cli would be in `./packages/cli/bin/run`.
```

Comment:
```suggestion
The cli version required to verify this proposal been released yet, but it can be found [in this branch](https://github.com/luisgj/celo-monorepo/tree/cREAL_cli). After pulling, it can be built by executing `yarn && yarn build` on the root, and then the binary for the cli would be in `./packages/cli/bin/run`.

A binary for this cli version can be found (here)[https://drive.google.com/file/d/10p-aMvNP77jvre6IlyyvCJWbMAl67avT/view?usp=sharing] (MacOS).
```